net/raft/http.go

```diff
@@ -161,6 +168,17 @@ func sendmsg(addr string, data []byte, client *http.Client) {
 	defer resp.Body.Close()
 }
 
+// Sends eviction notice to a node to kill the process
+func evictionNotice(addr string, client *http.Client) {
+	url := "https://" + addr + "/raft/kill"
+	resp, err := client.Post(url, contentType, nil)
+	if err != nil {
+		log.Printkv(context.Background(), "warning", err)
+		return
+	}
+	defer resp.Body.Close()
+}
```

Comment:
I wonder if instead of pushing eviction notices to evicted nodes, we should provide facilities for a node to recognize that it's evicted when it performs a `sendmsg` to contact other nodes?

Since these eviction notices are best-effort, it's still possible for an eviction to go unnoticed. For example:
1. Node _A_ is the Chain Core leader process.
2. A network partition causes node _A_ to be unreachable from the other processes.
3. An operator intervenes and evicts node _A_ from the cluster.
4. All the other nodes process the eviction entry and try to deliver the eviction notice to node _A_. Because node _A_ is unreachable, they all fail.
5. The network is restored.
6. Node _A_ is still evicted, but its Chain Core process is still running.

Node _A_ would continue to be Chain Core leader but without a working raft cluster.

I guess this scenario could also happen if the network partition is never restored and the node is never evicted. The leader process could continue despite not being able to participate in raft. Maybe the real solution is just to move Chain Core leadership into raft.